gnucash/gnome-utils/gnc-gnome-utils.h

```diff
@@ -68,6 +68,11 @@ void gnc_gnome_help (GtkWindow *parent, const char *file_name,
  */
 void gnc_launch_doclink (GtkWindow *parent, const char *uri);
 
+/** Retrun the first day of the week based on the following order.
+ *  Sunday = 1, M = 2, T = 3, W = 4, Th = 5, Fr = 6, Sat = 7
+ */
+gint gnc_get_first_day_of_week (void);
+
```

Comment:
This isn't needed, nor is the multi-platform implementation. `gnc_start_of_week` works everywhere, use that.

Reply:
@jralls I am not sure about this, just tested on my windows 10 machine.
If I change the start day under, `Control Panel->Clock and Region->Region` and set it to `Wednesday`...
My function returns 4 but `gnc_start_of_week` returns 2, I assuming that is the default for the region.

Reply:
@Bob-IT That's expected and consistent with all of the other localization in GnuCash. We use POSIX localization which doesn't support tailored locales.

I'm in favor of changing that&mdash;it's a source of periodic compliants&mdash;but it has to apply to all localization in GnuCash, which is way out of scope for this PR.

Reply:
OK.